pkgs/by-name/na/nagstamon/package.nix

```
@@ -0,0 +1,67 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  qt6Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "nagstamon";
+  version = "3.16.2";
+
+  src = fetchFromGitHub {
+    owner = "HenriWahl";
+    repo = "Nagstamon";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-9w8ux+AeSg0vDhnk28/2eCE2zYLvAjD7mB0pJBMFs2I=";
+  };
+
+  build-system = with python3Packages; [ setuptools ];
+
+  buildInputs = [
+    qt6Packages.qtmultimedia
+    qt6Packages.qtbase
+    qt6Packages.qtsvg
+    qt6Packages.qtwayland
+  ];
```

```suggestion
  nativeBuildInputs = [ qt6Packages.wrapQtAppsHook ];

  buildInputs = [
    qt6Packages.qtmultimedia
    qt6Packages.qtsvg
  ];
```

qtbase and qtwayland are propagated by the wrapper

---

Thanks for the tip. Fixed.